ivy/functional/backends/jax/creation.py

```diff
@@ -319,9 +319,9 @@ def one_hot(
     depth: int,
     /,
     *,
-    on_value: Optional[Number] = None,
```

Comment:
Given that this is inspired from `tf.one_hot`, not sure if we should change the default value in this case, but given that they've mentioned in their docs that they defer to `1` and `0` when they're `None`, probably this change is more sensible, as `axis=-1` is more intuitive and is also something `jax.nn.one_hot` does

Reply:
i think having explicit values makes more sense indeed even if the signature is sto-, i mean, taken, from tensorflow

Reply:
sounds good, thanks!